collector/pg_process_idle.go

```diff
@@ -111,7 +124,7 @@ func (PGProcessIdleCollector) Update(ctx context.Context, instance *instance, ch
 	}
 	secondsSumMetric := 0.0
 	if secondsSum.Valid {
-		secondsSumMetric = float64(secondsSum.Int64)
+		secondsSumMetric = float64(secondsSum.Float64)
```

Comment:
Simplification?

```suggestion
		secondsSumMetric = secondsSum.Float64
```

Reply:
I don't know go at all, never mind the library that is using but if `Float64` is just a normal float that doesn't need a cast then sure.

Reply:
Seems to work so I've made the change.